src/textual/command.py

```diff
@@ -598,23 +674,36 @@ def _stop_no_matches_countdown(self) -> None:
     _NO_MATCHES_COUNTDOWN: Final[float] = 0.5
     """How many seconds to wait before showing 'No matches found'."""
 
-    def _start_no_matches_countdown(self) -> None:
+    def _start_no_matches_countdown(self, search_value: str) -> None:
```

Comment:
Isn't it cleaner to just return without scheduling a no-op callback when there is no search value?

Reply:
I'm struggling to understand this question.